src/grpcbox_client_stream.erl

```diff
@@ -222,3 +212,20 @@ encoding_to_binary(deflate) -> <<"deflate">>;
 encoding_to_binary(snappy) -> <<"snappy">>;
 encoding_to_binary(Custom) -> atom_to_binary(Custom, latin1).
 
+headers(Scheme, Host, Path, Encoding, MessageType, MD) ->
+    {UserAgent, MD1} = case lists:search(fun({Header, _}) -> string:lowercase(Header) == <<"user-agent">> end, MD) of
```

Comment:
Since it is grpc which defines it as lowercase I think we can skip this and just use `lists:keystore` and not have to do a `search` + `replace`.

Reply:
@tsloughter ready